common/annotations/kab.xml

```diff
@@ -1299,8 +1295,6 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="✒" type="tts" draft="unconfirmed">imru n tricet aberkan</annotation>
 		<annotation cp="🖋" draft="unconfirmed">imru n lmidad</annotation>
 		<annotation cp="🖋" type="tts" draft="unconfirmed">imru n lmidad</annotation>
```

Comment:
> ...delete both the St. Martin & Sint Maarten annotations flag emojis in kab...
After their translation of St. Martin we could put (FR) and after Sint Maaren (NL). That at least gives them both a name, and the translators can fix next time.

Reply:
(^ agree, edited for clarity)

Reply:
Removed the duplicate values for St. Martin in regions.